samples/matter/window_covering/sample.yaml

```diff
@@ -2,33 +2,33 @@ sample:
   description: Matter Window Covering example
   name: Matter Window Covering
 tests:
+  # Excluded in quarantine.yaml to limit resources usage in integration builds
   sample.matter.window_cover.debug:
     build_only: true
+    extra_args: CONFIG_CHIP_LIB_SHELL=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
-  sample.matter.window_cover.no_dfu:
-    build_only: true
-    extra_args: CONF_FILE=prj_no_dfu.conf
-    integration_platforms:
-      - nrf21540dk_nrf52840
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840
-    tags: ci_build
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
   sample.matter.window_cover.release:
     build_only: true
     extra_args: CONF_FILE=prj_release.conf
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
+      - nrf7002dk_nrf5340_cpuapp
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
   sample.matter.window_cover.smp_dfu:
     build_only: true
-    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y
+    extra_args: CONFIG_CHIP_LIB_SHELL=y CONFIG_CHIP_DFU_OVER_BT_SMP=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
+  # ---------------
+  sample.matter.window_cover.no_dfu:
```

Comment:
Not sure if it's not too dangerous not to build any DFU variant for the sample. Especially if there's no nightly/weekly test that verifies all build types. It's likely that we miss some mcuBoot configuration etc.

Reply:
I see, so basically we need to change requirement to configuration selection from "Contains all supported Matter over Thread accessories" to "Contains all supported Matter over Thread accessories with DFU enabled", right?

Reply:
I think this might be a good idea.